src/wp-includes/formatting.php

```diff
@@ -3294,11 +3294,14 @@ static function ( $matches ) {
  *
  * @since 5.1.0
  * @since 5.6.0 Removed 'noreferrer' relationship.
+ * @deprecated 5.7.0
  *
  * @param string $text Content that may contain HTML A elements.
  * @return string Converted content.
  */
 function wp_targeted_link_rel( $text ) {
```

Comment:
Both this and `wp_targeted_link_rel_callback()` will need to be moved to `src/wp-includes/deprecated.php`

They can be added to the end of the file.

Reply:
> Both this and `wp_targeted_link_rel_callback()` will need to be moved to `src/wp-includes/deprecated.php`
> 
> They can be added to the end of the file.

This was incorrect. Deprecated functions in this file are retained in place as there may be situations in which this file is loaded but deprecated.php is not.